tests/contrib/molten/test_molten.py

```diff
@@ -0,0 +1,48 @@
+from unittest import TestCase
+
+import molten
+from ddtrace import Pin
+from ddtrace.contrib.molten import patch
+from molten import App, Route
+from molten.testing import TestClient
+from nose.tools import eq_, ok_
+
+from ...test_tracer import get_dummy_tracer
+
+
+class TestMolten(TestCase):
+    """"Ensures Molten is properly instrumented."""
+
+    TEST_SERVICE = 'molten-patch'
+
+    def setUp(self):
+        self.tracer = get_dummy_tracer()
+        # simple hello app
+        def hello(name: str, age: int) -> str:
+            return f"Hello {age} year old named {name}!"
+        self.app = App(routes=[Route("/hello/{name}/{age}", hello)])
+        patch()
+
+    def tearDown(self):
+        pass
+
+    def test_route_success(self):
+        client, tracer = self.get_client_and_tracer()
```

Comment:
using the default `Pin` for most tests cases is probably fine, we will want to have at least 1 that tests overriding the default `Pin`, but we don't need that for every test.